.env.circle-integration-w3t

```diff
@@ -8,10 +8,13 @@ GANACHE_HOST=0.0.0.0
 GANACHE_PORT=8545
 USE_GANACHE_DEPLOYMENT_CACHE = 'true'
 
-# For e2e-test
+# e2e-tests
 BROWSER_LOG_DESTINATION = 'browser.log'
-HEADLESS = 'true'
-USE_DAPPETEER=false
+HEADLESS='true'
+USE_DAPPETEER='false'
+TARGET_NETWORK = 'localhost'
+80faa6ee8'
+# ( etherlime account 0 )
```

Comment:
When this PR is ready for review, we probably need to clean this up and make it consistent with `.env.circle-integration-rps`.

Reply:
Thanks @junjizhi . This is partly a typo, fixed in 2906f31f2fd46bd2565ba850e737c5a0c5ea52c9. 

As for consistency with existing `.env.foo` files, I added `DAPPETEER_PK` to all the relevant ones (its a new env var as of this PR).

Reply:
Along with a comment about where this private key comes from. I put it on a separate line because we have had issues with trailing whitespace in these files.